src/routes/committee/suggestions/+page.svelte

```diff
@@ -75,7 +82,11 @@
 					/>
 				</StructuredListCell>
 				<StructuredListCell>
-					<Button kind="danger-tertiary" size="small" iconDescription="Like" icon={Favorite} />
+					{#if song.user.id == 'KcRkWMQUEClLEeiccSD5'}
+						<Button kind="danger-tertiary" size="small" iconDescription="Like" icon={Bat} />
+					{:else}
+						<Button kind="danger-tertiary" size="small" iconDescription="Like" icon={Favorite} />
+					{/if}
```

Comment:
Future maintainers be like "whyyyyyyy" kek

Reply:
:pepecute: